src/overlays/actors/ovl_Boss_06/z_boss_06.c

```diff
@@ -621,7 +621,7 @@ void Boss06_Draw(Actor* thisx, GlobalContext* globalCtx2) {
                         Matrix_InsertYRotation_f(M_PI, MTXMODE_APPLY);
                     }
 
-                    Matrix_Scale(-0.002f, -this->unk_1D8, 1.0f, MTXMODE_APPLY);
+                    Matrix_Scale(-0.0019999999f, -this->unk_1D8, 1.0f, MTXMODE_APPLY);
```

Comment:
Can this be written as some nice product like `2/1000`?

Reply:
Found it, its `0.02f / 10.0f`. Interesting to note that `0.02f` * `0.10f` is still off by 1.

Reply:
Yeah, I think @petrie911 had a conjecture about the ways the powers of 10 matter. I'm sure someone will do an audit of them all in future, but good to catch the easy ones now.